rocksdb_admin/admin_handler.cpp

```diff
@@ -53,6 +53,9 @@ DEFINE_int32(port, 9090, "Port of the server");
 DEFINE_string(shard_config_path, "",
              "Local path of file storing shard mapping for Aperture");
 
+DEFINE_bool(allow_global_seqno, false,
```

Comment:
nit: rename it to rocksdb_allow_global_seqno, and see if we can put them on the same line?